src/plugins/graph/hooks/use-data-tips.ts

```diff
@@ -1,73 +0,0 @@
-import {select} from "d3";
-import React, {useEffect} from "react";
-import {tip as d3tip} from "d3-v6-tip";
-import {IGraphModel} from "../models/graph-model";
-import {IDotsRef, transitionDuration} from "../graph-types";
-import {IDataSet} from "../../../models/data/data-set";
-import {getPointTipText} from "../utilities/graph-utils";
-import {RoleAttrIDPair} from "../models/data-configuration-model";
-import { CaseData } from "../d3-types";
-
-const dataTip = d3tip().attr('class', 'graph-d3-tip')/*.attr('opacity', 0.8)*/
-  .attr('data-testid', 'graph-point-data-tip')
-  .html((d: string) => {
-    return `<p>${d}</p>`;
-  });
-
-interface IUseDataTips {
-  dotsRef: IDotsRef,
-  dataset: IDataSet | undefined,
-  graphModel: IGraphModel,
-  enableAnimation: React.MutableRefObject<boolean>
-}
-export const useDataTips = ({dotsRef, dataset, graphModel, enableAnimation}:IUseDataTips) => {
-  const hoverPointRadius = graphModel.getPointRadius('hover-drag'),
-    pointRadius = graphModel.getPointRadius(),
-    selectedPointRadius = graphModel.getPointRadius('select'),
-    roleAttrIDPairs:RoleAttrIDPair[] = graphModel.config.uniqueTipAttributes,
-    yAttrIDs = graphModel.config.yAttributeIDs;
-
-  useEffect(() => {
-
-    function okToTransition(target: any) {
-      return !enableAnimation.current && target.node()?.nodeName === 'circle' && dataset &&
-        !target.property('isDragging');
-    }
-
-    function showDataTip(event: MouseEvent) {
-      const target = select(event.target as SVGSVGElement);
-      if (okToTransition(target)) {
-        target.transition().duration(transitionDuration).attr('r', hoverPointRadius);
-        const caseID = (target.datum() as CaseData).caseID,
-          plotNum = (target.datum() as CaseData).plotNum, // Only can be non-zero for scatter plots
-          attrIDsToUse = roleAttrIDPairs.filter((aPair) => {
-            return plotNum > 0 || aPair.role !== 'rightNumeric';
-          }).map((aPair) => {
-            return plotNum === 0
-              ? aPair.attributeID
-              : aPair.role === 'y' ? yAttrIDs[plotNum] : aPair.attributeID;
-          });
-        const tipText = getPointTipText(caseID, attrIDsToUse, dataset);
-        tipText !== '' && dataTip.show(tipText, event.target);
-      }
-    }
-
-    function hideDataTip(event: MouseEvent) {
-      const target = select(event.target as SVGSVGElement);
-      dataTip.hide();
-      if (okToTransition(target)) {
-        const caseID = (select(event.target as SVGSVGElement).datum() as CaseData).caseID,
-          isSelected = dataset?.isCaseSelected(caseID);
-        select(event.target as SVGSVGElement)
-          .transition().duration(transitionDuration)
-          .attr('r', isSelected ? selectedPointRadius : pointRadius);
-      }
-    }
-
-    dotsRef.current && select(dotsRef.current)
```

Comment:
The original code had mouseOver and mouseOut events such that the radius of the outer circle shrank when hovered over. Removing this meant `use-data-tips` was not actually providing any use to the code, so I removed the file.

Reply:
This is the code that provides tooltip-like information about the case/point being hovered over. I suspect you're going to want that at some point, even if you don't need it now.

Reply:
Also, that expand-on-hover feature is very useful when trying to identify individual points in a dense scatter plot.

Reply:
Got it. I brought back `use-data-tips.ts` and just commented out the invocation in `graph.tsx` with a comment.